helix-term/src/commands/typed.rs

```diff
@@ -1623,6 +1623,28 @@ fn run_shell_command(
     Ok(())
 }
 
+fn redraw(
+    cx: &mut compositor::Context,
+    _args: &[Cow<str>],
+    event: PromptEvent,
+) -> anyhow::Result<()> {
+    if event != PromptEvent::Validate {
+        return Ok(());
+    }
+
+    let callback = Box::pin(async move {
+        let call: job::Callback = Box::new(|_editor, compositor| {
+            compositor.clear().expect("unable to redraw");
```

Comment:
Not sure how OK it is to panic here. I was looking how the error from `terminal.clear()` is propagated and it ultimately panics on `resize()` so... :shrug: